### ISSUE TITLE
The Edge PoC should be based on some standard Open Data Hub demo / training

### ISSUE THREAD

**Issue by adelton** [opendatahub-io/ai-edge#136]
The https://github.com/opendatahub-io/ai-edge contains a PoC of setup that in the https://github.com/opendatahub-io/ai-edge/tree/main/pipelines part works primarily with the two models, builds container images, and does some "GitOps" actions.

The ACM part then shows extending exployment of these models as applications to ACM-managed clusters, a.k.a. near edge.

We have a couple of issues filed to extend the matrix of use-cases, for example https://github.com/opendatahub-io/ai-edge/issues/57 ... but for things that extend the non-ACM part of the PoC, shouldn't we either use existing Open Data Hub demos / training paths / example repos, or turn the majority of the pipelines part into one? Aren't we duplicating efforts?

If we used and contributed to an existing demo / training repo / example repo for the basic model handling, we would help improve example for people who do not necessarily care about edge, and this repo could then be a fork or an addition to the base one, focusing on the actual edge parts.

**Comment by LaVLaS** [comment]
I completely agree that we should be reusing existing demos and training paths for edge use cases as a form of dogfooding our own materials AND ensuring that the entire workflow stays relevant.  

> If we used and contributed to an existing demo / training repo / example repo for the basic model handling, we would help improve example for people who do not necessarily care about edge

IMO, we should be including edge use case in the same location as "existing demo / training repo / example repo for the basid model handling" to ensure that anyone learning about ODH can learn how to do basic model handling and/or distributed model handling (Edge).  One the discoveries from the PoC are included as features in the ODH components the workflows for basic & distributed model  handling should share the same user experience

**Comment by adelton** [comment]
> IMO, we should be including edge use case in the same location as "existing demo / training repo / example repo for the basid model handling" to ensure that anyone learning about ODH can learn how to do basic model handling and/or distributed model handling (Edge). One the discoveries from the PoC are included as features in the ODH components the workflows for basic & distributed model handling should share the same user experience

Once we consider the approach stable, with CI and other stuff, sure, let's offer it to the "upstream" demo / repo. But currently we have a bunch of unresolved bigs and high priority issues opened, and it would not be fair to polute the existing demo / repo with something that generates a high churn and has a high maintenance cost.

We can eventually merge the Edge cases in completely, but that shouldn't be the primary goal of this issue. Here I would propose to focus on finding what existing ODH demos there are and how to build on top of them first.
